### PR TITLE
add bulk tag screenshot

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: static/images/service_management/service_level_objectives/slo_bulk_tag.png
Comment (estherk15) [inline static/images/service_management/service_level_objectives/slo_bulk_tag.png:1]: Hi @roxanne-moslehi can you crop out the navigation. Also, if available in demo org, can you take a screenshot from there?

File: content/en/service_management/service_level_objectives/_index.md [hunk 1]
```diff
@@ -152,9 +152,12 @@ Sort SLOs by the *status* and *error budget* columns to prioritize which SLOs ne
 
 ### SLO tags
 
-When you create or edit an SLO, you can add tags for filtering on the [SLO status page][2] or for creating [SLO saved views][17].
+SLO Tags can be used for filtering on the [SLO status page][2], creating [SLO saved views][17], or grouping SLOs to view. Tags can be added to SLOs in the following ways:
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/_index.md:155]:
```suggestion
SLO tags can be used for filtering on the [SLO status page][2], creating [SLO saved views][17], or grouping SLOs to view. Tags can be added to SLOs in the following ways:
```

File: content/en/service_management/service_level_objectives/_index.md [hunk 2]
```diff
@@ -152,9 +152,12 @@ Sort SLOs by the *status* and *error budget* columns to prioritize which SLOs ne
...
 
-Add tags to SLOs in bulk with the *Edit Tags* and the *[Edit Teams][18]* dropdown options at the top of the SLO list.
+- When you create or edit an SLO, you can add tags
+- From the SLO list view, you can add/update tags in bulk using the *Edit Tags* and the *[Edit Teams][18]* dropdown options at the top of the SLO list:
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/_index.md:158]:
```suggestion
- From the SLO list view, you can add and update tags in bulk using the *Edit Tags* and the *[Edit Teams][18]* dropdown options at the top of the SLO list.
```

File: content/en/service_management/service_level_objectives/_index.md [hunk 3]
```diff
@@ -152,9 +152,12 @@ Sort SLOs by the *status* and *error budget* columns to prioritize which SLOs ne
...
+
+{{< img src="service_management/service_level_objectives/slo_bulk_tag.png" alt="SLO bulk tags" >}}
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/_index.md:160]:
```suggestion
{{< img src="service_management/service_level_objectives/slo_bulk_tag.png" alt="SLO list page displays the Edit Tag dropdown for bulk tag editing" >}}
```

File: static/images/service_management/service_level_objectives/slo_bulk_tag.png
Comment (roxanne-moslehi) [reply static/images/service_management/service_level_objectives/slo_bulk_tag.png:1]: @estherk15 this is from the demo org, and do you mean the side nav?

**Review by estherk15 — APPROVED** [review]

**Pull request merged**